### PR TITLE
Add search UI to API docs

### PR DESCRIPTION
**Pull request by ScottRudiger**
To increase developer productivity, we're adding a search feature that will return results from across our developer site.

### CODE REVIEW

File: source/layouts/layout.erb [hunk 1]
```diff
@@ -31,6 +31,11 @@ under the License.
     <%= stylesheet_link_tag :screen, media: :screen %>
     <%= stylesheet_link_tag :print, media: :print %>
     <link href="https://cdn.jsdelivr.net/flag-icon-css/1.1.0/css/flag-icon.min.css" rel="stylesheet" type="text/css">
+
+    <link
```
Comment (fastdivision) [inline source/layouts/layout.erb]: @ScottRudiger Can you make this stylesheet `link` tag a single line and place it right under line 33 (no newline above)?

File: source/layouts/layout.erb [hunk 2]
```diff
@@ -162,5 +168,23 @@ under the License.
...
+    <script src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+    <script>
+      docsearch({
+        appId: '7ZBSP80X0K',
+        apiKey: '2a9d77b7d7b611725a62fd9ffb522880',
+        indexName: 'developers.taxjar.com',
+        inputSelector: '#docsearch'
+      });
+    </script>
+    <script>
```
Comment (fastdivision) [inline source/layouts/layout.erb]: @ScottRudiger Do we need two `docsearch` calls here? Can we update `inputSelector` to `#docsearch, #docsearch-mobile` with the comma? If not, maybe we include both calls in the same `script` tag?

File: source/stylesheets/screen.css.scss [hunk 3]
```diff
@@ -248,8 +249,6 @@ a {
 
 .tocify-wrapper {
   @include transition(left ease-in-out 0.3s);
-  overflow-y: auto;
```
Comment (fastdivision) [inline source/stylesheets/screen.css.scss]: @ScottRudiger Can you share why we're removing `overflow-y` and `overflow-x` here?

File: source/layouts/layout.erb (hunk 2 above)
Comment (ScottRudiger) [reply source/layouts/layout.erb]: There is a [feature in the works](https://github.com/algolia/docsearch/issues/230) to allow multiple `inputSelector`, but for now we'll have to make two calls to `docsearch`. I've combined them into one `script` tag though.

File: source/stylesheets/screen.css.scss (hunk 3 above)
Comment (fastdivision) [reply source/stylesheets/screen.css.scss]: Thanks for following up and answering this!

**Comment by fastdivision** [comment]
@ScottRudiger I believe my commit to update the CSS was lost after a force push here.

**Comment by ScottRudiger** [comment]
Sorry about that @fastdivision, should've been more careful with the force push. I've recreated the CSS updates and also added a fix for the green highlights that weren't showing after the move to `docsearch.scss`.

File: source/stylesheets/screen.css.scss (hunk 3 above)
Comment (ScottRudiger) [reply source/stylesheets/screen.css.scss]: For posterity, we resolved this internally. The `overflow-x` and `overflow-y` properties did not seem to affect any other visual aspect and removing them fixed the issue with the DocSearch dropdown hiding behind other content.

**Pull request merged**